src/udp_wrap.h

```diff
@@ -46,7 +46,8 @@ class UDPListener {
   // Called right after data is received from the socket, and includes
   // information about the source address. If `nread` is negative, an error
   // has occurred, and it represents a libuv error code.
-  virtual void OnRecv(ssize_t nread,
+  virtual void OnRecv(uv_udp_t* handle,
```

Comment:
This doesn’t seem like something we should be doing … currently, `UDPListener` is already associated with one `UDPWrapBase` instance, so this should always be same same value and always retrievable

As far as I can tell, this is currently only being used for the `SocketAddress::FromSockName()` call above? I might be missing something, but that one could probably use `udp_->GetSockName()` instead?

Reply:
Ok, will rework it!